cmd/buildah/import.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/containers/storage/pkg/archive"
```

Comment:
I think you're going to want to use the neighboring `chrootarchive` package here.